android/android_browser_tests.gni

```diff
@@ -24,7 +24,7 @@ android_test_exception_sources = [
   "//brave/browser/brave_wallet/brave_wallet_sign_message_browsertest.cc",
   "//brave/browser/brave_wallet/brave_wallet_tab_helper_browsertest.cc",
   "//brave/browser/brave_wallet/ethereum_provider_browsertest.cc",
-  "//brave/browser/brave_wallet/send_transaction_browsertest.cc",
```

Comment:
If possible can you make them run on Android too? We aim to minimize this list as much as possible so Android will be covered.